tests/common/osutil/test_default.py

```diff
@@ -657,16 +658,20 @@ def set_command(command, output='', exit_code=0):
         wait = FlexibleVersion(version) >= osutil._IPTABLES_LOCKING_VERSION
         uid = 42
 
-        version_command = set_command(osutil._get_iptables_version_command(), output=str(version))
-        list_command = set_command(osutil._get_firewall_list_command(wait), output="Mock Output")
-        set_command(osutil._get_firewall_packets_command(wait))
-        set_command(osutil._get_firewall_drop_command(wait, "-C", destination))
-        set_command(osutil._get_firewall_drop_command(wait, "-A", destination))
-        set_command(osutil._get_firewall_accept_command(wait, "-A", destination, uid))
+        version_command = set_command(osutil.get_iptables_version_command(), output=str(version))
+        list_command = set_command(osutil.get_firewall_list_command("-w" if wait else ""), output="Mock Output")
```

Comment:
Not sure where my comment got lost, but since you're using the `"-w" if wait else ""` in every line, can you make it a variable to keep the code clean? Thanks!

Reply:
Addressed